wcomponents-theme/src/test/intern/wc.dom.getLabelsForElement.test.js

```diff
@@ -66,6 +66,45 @@ define(["intern!object", "intern/chai!assert", "./resources/test.utils!"],
 				var result = controller(document.getElementById("unlabelled"));
 				assert.isTrue(Array.isArray(result));
 				assert.strictEqual(0, result.length);
+			},
+			testAriaLabel: function() {
+				var element = document.getElementById("aria-text");
```

Comment:
We tend to use a single `var`. This is just project convention and in the world of JS lots of excellent coders don't use a single var, it's not "good" or "bad" either way (although it's fewer characters to use a single var :) ). You don't have to change it (though I think Mark's planned ESlint rule tweak might make this break, not sure).

Reply:
Sure, I can conform to project convention. I using multiple var was out of java code style.